coordinator/src/bin/coordinator.rs

```diff
@@ -162,6 +163,18 @@ async fn main() -> Result<()> {
         }
     });
 
+    tokio::spawn({
+        let node = node.clone();
+        async move {
+            loop {
+                tokio::time::sleep(UNREALIZED_PNL_SYNC_INTERVAL).await;
+                if let Err(e) = closed_positions::sync(node.clone()) {
```

Comment:
🤔 Is that a copy paste error? Shouldn't it call `unrealized_pnl::sync` here?

Reply:
😅 omg, good catch!